amundsen_application/static/js/components/TableDetail/ColumnListItem/index.tsx

```diff
@@ -59,7 +59,9 @@ export class ColumnListItem extends React.Component<
         label: `${metadata.name} ${metadata.col_type}`,
       });
     }
-    this.setState({ isExpanded: !this.state.isExpanded });
+    if (this.shouldRenderDescription() || this.props.data.stats.length !== 0) {
```

Comment:
Nit: I would move the line 55 up one position and use here `|| metadata.stats.length`